.github/workflows/django.yml

```
@@ -53,12 +37,12 @@
             inject-env-vars: true
-        - name: Run Migrations
-          run: |
-            task db_sync
-
         - name: Run Tests
-          run: |
-                export DJANGO_CONFIGURATION=Development && export DOPPLER_TOKEN=${{ secrets.DOPPLER_TOKEN }} 
-                task test
-  
\ No newline at end of file
+          run: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)